cli/src/main.rs

```diff
@@ -463,6 +482,53 @@ async fn run_cache(command: &CacheCommand) -> Result<(), CliError> {
     Ok(())
 }
 
+async fn run_init(args: &InitArgs) -> Result<(), CliError> {
+    let template_json = include_str!("templates.json");
+    let mappings: HashMap<String, Vec<String>> = serde_json::from_str(template_json)?;
+    let language = args.language.to_lowercase().trim().to_owned();
+
+    let Some(url) = mappings
+        .iter()
+        .find_map(|(k, v)| v.contains(&language).then_some(k)) else {
+            return Err(CliError::TemplateTag(language));
+        };
+
+    let Some(folder_name) = url.rsplit_once('/').map(|(_, second)| second) else {
+            return Err(CliError::TemplateTag(language));
+        };
+
+    std::process::Command::new("git")
+        .arg("clone")
+        .arg(url)
+        .output()?;
+
+    let project_path = fs::canonicalize(folder_name)?;
+
+    let mut git_path = project_path.clone();
+    git_path.push(".git");
+
+    let mut gitignore_path = project_path.clone();
+    gitignore_path.push(".gitignore");
+
+    remove_dir_all(&git_path)?;
+    remove_file(&gitignore_path)?;
+
+    std::process::Command::new("rm")
+        .current_dir(&project_path)
+        .arg("-rf")
+        .arg(".git")
+        .output()?;
+
+    std::process::Command::new("rm")
+        .current_dir(&project_path)
+        .arg(".gitignore")
+        .output()?;
+
+    println!("Package template successfully intitialised!");
```

Comment:
Maybe point to the path, like `Package template created in the folder cpp_template`? Since you already have the folder name available